defaults/main.yml

```diff
@@ -22,18 +13,26 @@ react_git_key_filename: "id_ed25519"
 react_remove_git_key: true
 
 # system-wide dependencies
-react_system_wide_dependencies:
+react_remote_system_wide_dependencies:
   - yarn
+  - git
+react_local_system_wide_dependencies:
+  - git
 react_node_version: 10.x
 
 # app
+react_remote_js_build: True
+react_local_checkout_path: "{{ react_local_user_home }}/reveal"
```

Comment:
Could we have the local checkout in a temporary directory that potentially gets removed after the build process? Specifically in the `/tmp/$USER/reveal` directory?
A git clone in `/tmp` folder is the approach taken in the ansible-superset role, trying to understand why the home folder is an option here.
Or perhaps, `~/.cache/reveal` folder if caching is the intention.

Reply:
I prefer `/tmp` over `.cache`.

Reply:
cool.